src/components/Home/Home.tsx

```diff
@@ -27,18 +41,67 @@ export function Home() {
       <br />
     </>
   ) : null;
+
+  const handleSubmit = () => {
+    const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
+    if (postalCodeRegex.test(postalCode)) {
+      setRenderCards(true);
+      setShowPostalPrompt(false);
+    } else {
+      setShowInvalidPostal(true);
+    }
+  };
+  const afterPostalEntryMarkup = renderCards ? (
+    <>
+      <TextStyle variation="subdued">
+        Here&apos;s what&apos;s happening with vaccinations in Ottawa
+      </TextStyle>
+      <br />
```

Comment:
@moni-li curious abt your thoughts re: <br/> for styling w.r.t. accessibility. my understanding is that <br>s are used for semantic breaks in text

Reply:
> @moni-li curious abt your thoughts re: `<br/>` for styling w.r.t. accessibility. my understanding is that `<br/>`s are used for semantic breaks in text

The ideal is to not use `<br />` for stylistic purposes - e.g. to add whitespace

Reply:
Removed All `<br />` tags.